Fix a race condition that causes aclocal7 to fail on fast machines (see the
comment below).

diff -x config.log -x config.status -ru tmp/automake-1.12.2/t/aclocal7.sh work/automake-1.12.2/t/aclocal7.sh
--- tmp/automake-1.12.2/t/aclocal7.sh	2012-07-08 11:48:25.000000000 +0100
+++ work/automake-1.12.2/t/aclocal7.sh	2012-07-10 11:58:50.325999591 +0100
@@ -45,6 +45,11 @@
   $AUTOMAKE --no-force
 }

+# aclocal will rewrite aclocal.m4 unless the input files are all older than the
+# existing aclocal.m4 -- sleep to ensure somedefs.m4 has an older timestamp
+# than the aclocal.m4 that the next aclocal call will generate.
+$sleep
+
 $ACLOCAL -I m4
 AUTOMAKE_after_aclocal
